migrations/src/postDeploymentSetup.js

```diff
@@ -44,23 +44,25 @@ module.exports = async () => {
   // Only transfer tokens in testnets
   if (NETWORK !== 'mainnet') {
     // Add new instance of StakeManager contract & Deployer address as Minter
-    await schellingCoin.addMinter(stakeManagerAddress);
+
     await schellingCoin.addMinter(signers[0].address);
```

Comment:
This line is not needed IMO
If deployed new, signers[0] is minter only my constructor
If reusing we are adding him as minter at line
https://github.com/razor-network/contracts/pull/130/files#diff-268ac3aa71a022d9f10b6c5603857967f9f237b5f58ff6a1569ea63a2f2141c8R56

Reply:
@abhishekvispute I have made changes as per the discussion. I have resolved issue #141 too.

Reply:
Hey @adi44 
Please remove grant role as well from shelling coin as we discussed.
And also Please update PR title and desc for same